website/docs/guides/markdown-features/markdown-features-code-blocks.mdx

```diff
@@ -113,6 +114,12 @@ export default {
 
 After adding `additionalLanguages`, restart Docusaurus.
 
+:::warning
+
+When using Java or Scala, both must be included together in additionalLanguages
+
+:::
```

Comment:
This is true, but this doesn't only apply to Java/Scala, and other related languages are affected as well.

See also past issues
- https://github.com/facebook/docusaurus/issues/7209
- https://github.com/facebook/docusaurus/issues/5443
- https://github.com/facebook/docusaurus/issues/5013

Do we really want to add a documentation like this for something that is non-exhaustive, and can eventually be handled with code automatically?